### DISCUSSION TITLE
abreviated revision size

### DISCUSSION THREAD

**Discussion by jschaeff** [magit/magit#4521] [Support] [answered]
Hello,

is there a way to change the size of the abreviated revision ?

Magit is showing the 7 first chars of the SHA1, but other tools use 8 (for instance gitlab). So I would like Magit to show me the same.

**Comment by tarsius** [comment, marked as answer]
Set the git variable `core.abbrev`.

**Comment by jschaeff** [comment]
Excellent ! So simple I feel (even more) stupid.
